spec/bot_spec.rb

```diff
@@ -6,8 +6,8 @@
 
 describe Bot do
   let(:action)                { "created" }
-  let(:title)                 { "[#LIEF-1234] Cure world hunger!" }
-  let(:branch_name)           { "feature/lief-1234-cure-world-hunger!" }
+  let(:title)                 { "[LIEF-1234] Cure world hunger!" }
+  let(:branch_name)           { "feature/LIEF-1234-cure-world-hunger!" }
```

Comment:
I suspect, in the real world the exclamation mark would get removed by tickety-tick 🙂